README.md

```diff
@@ -34,31 +34,54 @@ There are two Senders:
 
 3. Create a `.env` file: `cp .env.example .env` and fill in the information using the output data from previous deploy step. Check the `.env.example` file to see the required variables.
 
-4. To run the processes:
-  - `npm run watcher:deposit`
+## Run the processes
+
+### Native to Erc mode
+  - `npm run watcher:signature-request`
   - `npm run watcher:collected-signatures`
-  - `npm run watcher:withdraw`
+  - `npm run watcher:affirmation-request`
+  - `npm run sender:home`
+  - `npm run sender:foreign`
+
+To send deposits to home contract run `node tests/sendUserTxToHome.js`
+
+To send withdrawals to foreign contract run `node tests/sendUserTxToForeign.js`
+
+Make sure your `HOME_MIN_AMOUNT_PER_TX` and `FOREIGN_MIN_AMOUNT_PER_TX` is same as in your .env deployment contract
+
+
+### Erc to Erc mode
+  - `npm run watcher:erc:signature-request`
+  - `npm run watcher:erc:collected-signatures`
+  - `npm run watcher:erc:transfer`
```

Comment:
should we introduce new watcher `erc:transfer`? Can `affirmation-request` depending on the bridge mode listen to correct events?

Reply:
Yes, I think you are right, I will change the name of the script to `watcher:erc:affirmation-request` to avoid introducing new watcher names. Thanks